lib/utils/keybuilder/src/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const dbClient = require('abacus-dbclient');
+
+module.exports.createMeterDocId = (usageDoc) => {
```

Comment:
Probably we can rename the module to `docid` and have this function as`meterId` or `createMeterId`?

Reply:
I agree.